.github/workflows/test_all_in_one_common.yml

```diff
@@ -0,0 +1,90 @@
+name: test-all-in-one-common
+on:
+  workflow_call:
+    inputs:
+      secondary_tests:
+        required: true
+        type: string
+env:
+  UYUNI_PROJECT: uyuni-project
+  UYUNI_VERSION: master
+  CUCUMBER_PUBLISH_TOKEN: ${{ secrets.CUCUMBER_PUBLISH_TOKEN }}
+jobs:
+  test-uyuni:
+    runs-on: ubuntu-latest
+    # create a PR to add yourself if you want to be a beta user and
+    # run this action in your Pull Requests
+    if: github.actor == 'jordimassaguerpla' || github.actor == 'elariekerboull' ||  github.actor == 'srbarrios'
```

Comment:
For now, it's fine.
But long term, it could be better extract this list in a different file.

Reply:
It will be removed anyway, given this is only for the beta phase.